mantidproject/mantid
# DIFC calibration diagnostic plot

**Description of work.**

Adds the `difc_plot2d` function to `Calibration.tofpd.diagnostics` to calculate the change in DIFC between two instrument calibrations.

**To test:**

Run the examples seen in the docs to verify that generating the DIFC plot for each kind of input works properly.


Version into `ornl-next` #30810

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
